etl-pipeline/services/grin/download.py

```diff
@@ -0,0 +1,106 @@
+# Script run daily to download GRIN books that were converted in the past day
+
+from datetime import datetime, timedelta
+from typing import List
+from model import GRINState, GRINStatus, Record
+from managers import DBManager, S3Manager
+from logger import create_log
+from sqlalchemy import select, desc
+from .util import chunk
+from .grin_client import GRINClient
+
+logger = create_log(__name__)
+
+S3_BUCKET_NAME = "drb-grin-files-test"
+BATCH_SIZE_LIMIT = 1000
+
+
+class GRINDownload:
+    def __init__(self):
+        self.s3_client = S3Manager()
```

Comment:
recommend changing this to store_manager or s3_manager since there is a client which is the S3 client that can be referenced from this object